sycl/test/warnings/sycl_2020_deprecations.cpp

```diff
@@ -134,10 +134,10 @@ int main() {
   sycl::byte B;
   (void)B;
 
-  // expected-warning@+1{{'max_constant_buffer_size' is deprecated: max_constant_buffer_size is deprecated}}
+  // expected-error@+1{{no member named 'max_constant_buffer_size' in 'sycl::info::device'}}
```

Comment:
According to the test name the test checks deprecation warning. There is no reason to check here for missing API.
If we really need such check it should be a different test (I am not sure that we need it).